CHANGES.rst

```diff
@@ -89,6 +89,8 @@ Features added
 Bugs fixed
 ----------
 
+* #12543: Fix :pep:`695` formatting for LaTeX output.
+  Patch by Bénédikt Tran.
```

Comment:
This should go at the end of the list

Reply:
Oh I wasn't sure which was the last and which was the first :')